src/main/java/com/avairebot/AvaIre.java

```diff
@@ -277,6 +278,10 @@ public AvaIre(Settings settings) throws IOException, SQLException, InvalidApplic
         log.info("Preparing I18n");
         I18n.start(this);
 
+         log.info("Creating rank backgrounds");
+         RankBackgroundHandler rankBackgroundHandler = new RankBackgroundHandler();
+         rankBackgroundHandler.start();
```

Comment:
If we're not storing the `RankBackgroundHandler` in the Ava class with a getter method, and all the methods on the class are static anyways, the class should either not need to be instantiated, or be converted to a singleton.

Reply:
I needed it to have an constructor so I can make sure the background folder can be created. The best option is probably for it to be a singleton.

Reply:
Sounds good, just need to convert the file to a singleton then.